apps/meteor/tests/e2e/utils/test.ts

```diff
@@ -71,7 +71,7 @@ export const test = baseTest.extend<BaseTest>({
 				return request.put(BASE_API_URL + prefix + uri, { headers, data });
 			},
 			delete(uri: string, prefix = API_PREFIX) {
-				return request.delete(BASE_API_URL + prefix + uri, { headers });
+				return request.delete(BASE_URL + prefix + uri, { headers });
```

Comment:
`BASE_API_URL` is actually set like this:
```
export const BASE_API_URL = BASE_URL + API_PREFIX;
```

so I guess you should either remove `BASE_API_URL` , or keep using it and remove `prefix` from here..

btw, this needs to be changed for all operations

Reply:
So I went with the first approach & removed `BASE_API_URL` altogether. Also modified it for the other methods :smile: